pkg/component/types.go

```diff
@@ -40,7 +40,32 @@ type ComponentStatus struct {
 	LinkedServices   []string            `json:"linkedServices,omitempty"`
 }
 
-// State represents component state
+// CombinedComponentList is list of s2i and devfile components
+type CombinedComponentList struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ListMeta   `json:"metadata,omitempty"`
+	S2IComponents     []Component        `json:"s2i_components"`
```

Comment:
this is invalid json standardization.

It should be s2iComponents

Reply:
done